code/modules/hallucinations/effects/blind_rush_hallucination.dm

```diff
@@ -0,0 +1,68 @@
+/**
+  * # Hallucination - Blind Rush
+  *
+  * Makes target blind and causes them to see semi-transparent humanoids running at them.
+  */
+/datum/hallucination_manager/blind_rush
+	initial_hallucination = /obj/effect/hallucination/no_delete/blind_rusher
+	trigger_time = 34 //total lenght of the hallucination is a little more than ten seconds
```

Comment:
minor spelling error upon ye

`total lenght`

Reply:
Thank you! Didn't catch that!